src/NuGet.Core/NuGet.Frameworks/FrameworkConstants.cs

```diff
@@ -180,6 +189,12 @@ public static class CommonFrameworks
             public static readonly NuGetFramework NetCoreApp30
                 = new NuGetFramework(FrameworkIdentifiers.NetCoreApp, new Version(3, 0, 0, 0));
             public static readonly NuGetFramework NetCoreApp31
-                = new NuGetFramework(FrameworkIdentifiers.NetCoreApp, new Version(3, 1, 0, 0));        }
+                = new NuGetFramework(FrameworkIdentifiers.NetCoreApp, new Version(3, 1, 0, 0));
+            public static readonly NuGetFramework NetCoreApp50
+                = new NuGetFramework(FrameworkIdentifiers.NetCoreApp, Version5);
+
+            // .NET 5.0 and later has NetCoreApp identifier
+            public static readonly NuGetFramework Net50 = new NuGetFramework(FrameworkIdentifiers.NetCoreApp, Version5);
```

Comment:
just curious to know why `Net50` and `NetCoreApp50` has same identifier.

Reply:
net5.0 is being treated mostly as netcoreapp5.0.